executor/ddl.go

```diff
@@ -194,6 +198,15 @@ func (e *DDLExec) executeDropTable(s *ast.DropTableStmt) error {
 			return errors.Trace(err)
 		}
 
+		if config.CheckBeforeDrop {
+			log.Warnf("admin check table before drop was on")
```

Comment:
It's better to add more info in this log, e.g. the table name.

Reply:
done.